tools/uuid-corrector/src/uuid_corrector/core.clj

```diff
@@ -0,0 +1,137 @@
+(ns uuid-corrector.core
+  (:gen-class)
+  (:require [clojure.pprint :refer [pprint]]
+            [clojure.java.jdbc :as jdbc]
+            [clojure.string :as string]
+            [clojure.tools.cli :as cli]
+            [clojure.tools.logging :as log]
+            [honeysql.core :as sql]
+            [honeysql.helpers :refer :all]
+            [kameleon.pgpass :as pgpass]))
+
+(def ^:private cli-options
+  [["-h" "--host HOST" "Database host name"
+    :default "localhost"
+    :validate [(complement string/blank?) "Must not be blank."]]
+   ["-p" "--port PORT" "Database port number"
+    :default 5432
+    :parse-fn #(Integer/parseInt %)
+    :validate [#(< 0 % 65536) "Must be an integer between 0 and 65536."]]
+   ["-U" "--user USER" "Database username"
+    :default "de"
+    :validate [(complement string/blank?) "Must not be blank"]]
+   ["-d" "--de-database DATABASE" "DE database name"
+    :default "de"
+    :validate [(complement string/blank?) "Must not be blank"]]
+   ["-m" "--metadata-database DATABASE" "Metadata database name"
+    :default "metadata"
+    :validate [(complement string/blank?) "Must not be blank"]]
+   ["-?" "--help" "Display this help message"]])
+
+(defn- bad-usage
+  [summary errors]
+  (println "Invalid Usage:")
+  (dorun (map (comp println (partial str "\t")) errors))
+  (println)
+  (println summary)
+  (System/exit 1))
+
+(defn- show-help
+  [summary]
+  (println summary)
+  (System/exit 0))
+
+(defn- prompt-for-password
+  "Prompts the user for a password."
+  [user]
+  (print user "password: ")
+  (flush)
+  (.. System console readPassword))
+
+(defn- get-password
+  "Attempts to obtain the database password from the user's .pgpass file.  If
+  the password can't be obtained from .pgpass, prompts the user for the
+  password"
+  [host port database user]
+  (let [password (pgpass/get-password host port database user)]
+    (if (nil? password)
+      (if (nil? (System/console))
+        (binding [*out* *err*]
+          (println "No password supplied.")
+          (System/exit 1))
+        (prompt-for-password user))
+      password)))
+
+(defn- build-spec
+  [host port db user password]
+  {:subprotocol "postgresql"
+   :subname     (str "//" host ":" port "/" db)
+   :user        user
+   :password    password})
+
+(def uuid-columns [:created_by :modified_by])
+(def tables [:templates :attributes])
+(def table-columns (partition 2 (flatten (map (fn [table] (map #(list table %) uuid-columns)) tables))))
```

Comment:
This could also be accomplished with a `mapcat`:
```clojure
(def table-columns (mapcat (fn [table] (map #(list table %) uuid-columns)) tables))
```
I think this is a little more readable since it's a little shorter.